index.bs

```diff
@@ -1693,6 +1766,114 @@ The [=remote end steps=] with |command parameters| are:
 
 </div>
 
+#### The browsingContext.navigate Command ####  {#command-browsingContext-navigate}
+
+The <dfn export for=commands>browsingContext.navigates</dfn> command navigates a
+browsing context to the given URL.
+
+<dl>
+   <dt>Command Type</dt>
+   <dd>
+      <pre class="cddl remote-cddl">
+      BrowsingContextGetTreeCommand = {
+        method: "browsingContext.navigate",
+        params: BrowsingContextNavigateParameters
+      }
+
+      BrowsingContextNavigateParameters = {
+        context: BrowsingContext,
+        url: text,
+        ?wait: ReadinessState,
+      }
+
+       ReadinessState = "none" / "interactive" / "complete"
+      </pre>
+   </dd>
+   <dt>Return Type</dt>
+   <dd>
+    <pre class="cddl local-cddl">
+        BrowsingContextNavigateResult = {
+            navigation: Navigation / null,
+            url: text,
+        }
+    </pre>
+   </dd>
+</dl>
+
+<div algorithm="remote end steps for browsingContext.navigate">
+The [=remote end steps=] with |command parameters| are:
+
+  1. Let |context id| be the value of the <code>context</code> field of
+     |command parameters|.
+
+  1. Let |context| be the result of [=trying=] to [=get a browsing context=]
+     with |context id|.
+
+  1. Assert: |context| is not null.
+
+  1. Let |wait condition| be the value of the <code>wait</code> field of |command
+     parameters| if present, or "<code>none</code>" otherwise.
+
+  1. Let |url| be the value of the <code>url</code> field of |command
+     parameters|.
+
+  1. TODO: validate |url|. Should we allow relative URLs or only absolute URLs?
```

Comment:
CDP's Page.navigate requires an absolute URL at the moment, so this would require some protocol changes.

Reply:
So for now I've allowed relative URLs because this seems to reduce the burden on clients. But if that's going to be an implementation problem then we can certainly disallow them.